src/role-access/containers/exclusion/exclusion-navigation/exclusion-navigation.component.scss

```diff
@@ -0,0 +1,7 @@
+.govuk-back-link {
```

Comment:
Can we use one of the GDS classes `govuk-!-margin-bottom-1`

Reply:
this is not only add a margin-bottom, this is telling the app to inherit all properties from .govuk-back-link and override(customize) the margin bottom for our scenario.

Reply:
Hmmm but the `.govuk-back-link` class is used only in `exclusion-navigation.component.html`, we could just add an extra class `<a class="govuk-back-link govuk-!-margin-bottom-1" href="javascript:void(0)"></a>`.

Reply:
that is a fair point, the problem to concatenate the css in html is it will be a long statement if we have many additional css to override, but given this is just two styles I adopt your suggestions. please review again.